tests/unit/players-serializer.test.js

```diff
@@ -0,0 +1,44 @@
+const appRoot = require('app-root-path');
+const chai = require('chai');
```

Comment:
@nosnatef You are right that the serialize methods are hard to make fail. I did notice that `playerConverter` can fail though and should have its own tests.